airbyte-integrations/connectors/source-tiktok-marketing/manifest.yaml

```diff
@@ -327,6 +327,9 @@ definitions:
           filtering: '{{ {"secondary_status": "AD_STATUS_ALL"}|string if config.get("include_deleted", False) }}'
       record_selector:
         $ref: "#/definitions/record_selector"
+        record_filter:
```

Comment:
it feels a little weird to be filtering out these types of records on our side, but unfortunately I don't know of a better solution either so unfortunately we're probably left with this.

Let's add a comment with the proper context as to why we need to do this otherwise I'm sure we'll be confused why this had to be added in or if we want to eventually remove it.